grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -25,6 +25,93 @@ import spock.lang.Unroll
 class GrabbitConfigurationSpec extends Specification {
 
     @Unroll
+    def "Should return exclude paths"() {
+        when:
+        def output = GrabbitConfiguration.create(input)
+
+        then:
+        output instanceof GrabbitConfiguration
+
+        where:
+        input << [
+                """
+            {
+                "serverUsername" : "admin",
+                "serverPassword" : "admin",
+                "serverHost" : "localhost",
+                "serverPort" : "4503",
+                "deltaContent" : false,
+                "pathConfigurations" :  [
+                    {
+                        "path" : "/content/a/b",
+						"excludePaths" : ["/a", "\\b", "./c"],
```

Comment:
#23 says that absolute paths are explicitly illegal for exclusion. Why doesn't this blow up?